jormungandr/src/metrics/backends/simple_counter.rs

```diff
@@ -97,6 +104,10 @@ impl MetricsBackend for SimpleCounter {
         self.tx_recv_cnt.fetch_add(count, Ordering::SeqCst);
     }
 
+    fn set_tx_pending_cnt(&self, count: usize) {
+        self.tx_pending_cnt.store(count, Ordering::SeqCst);
```

Comment:
Idly wondering if this needs to be `SeqCst` if you fetch the metrics with `Relaxed`...

This shouldn't matter much in the grand scheme of things, though.

Reply:
I can revisit ordering here. I remember having odd test failures with less strict ordering requirements. Basically, I just copy-pasted these from older implementation.